### ISSUE TITLE
Home page example doesn't work as described

### ISSUE THREAD

**Issue by eugenesvk** [pest-parser/site#46]
upd: the website Editor should try to pick the semantically top-level rule instead of the first in an alphabetically sorted order, or if it's not possible, at least do this for the example from the homepage

Or maybe the easier option is to just document that the user needs to select the general rule manually

**Describe the bug**

I was trying out the example from the home page and instead of getting the error mentioned there got a working `- alpha: "a"`

**To Reproduce**
1. add the grammar from the home page
```
alpha = { 'a'..'z' | 'A'..'Z' }
digit = { '0'..'9' }

ident = { (alpha | digit)+ }

ident_list = _{ !digit ~ ident ~ (" " ~ ident)+ }
          // ^
          // ident_list rule is silent (produces no tokens or error reports)
```
2. paste `ab *`

**Expected behavior**

Get the error message
```
thread 'main' panicked at ' --> 1:1
  |
1 | ab *
  |    ^---
  |
  = expected ident', src/main.rs:12

```

**Actual behavior**

Happy parsing outcome `- alpha: "a"`

**Additional context**

**Comment by tomtau** [comment]
Yeah, it's not possible to tell which rule is the top-level one, but perhaps the homepage editor could be pre-filled with that example. @nathanielknight wdyt?

**Comment by HoloTheDrunk** [comment]
> Oh, I see, so it just doesn't select the most general rule by default, but defaults to the alphabetic sorting

Fairly certain it's actually just the one that's at the top, not the first in alphabetical order

**Comment by nathanielknight** [comment]
@eugenesvk Thanks for the report; I'm sure you're not the only person who's been caught out by this UX issue.

I think adding this as a default example (perhaps with an explanatory comment) would be an improvement.
